app/models/post.rb

```diff
@@ -4,6 +4,7 @@ class Post < ApplicationRecord
   has_many :likes, class_name: 'Like', foreign_key: 'post_id'
```

Comment:
- [ Required ] It would be better that when a `post` is deleted all dependent entries in the `likes` and `comments` are also deleted. I faced the error below when trying to delete a `post` that had `references` in the `likes` table. You can fix this by refactoring the following lines of code in your model `post.rb`

```ruby
  has_many :comments, class_name: 'Comment', dependent: :destroy, foreign_key: 'post_id'
  has_many :likes, class_name: 'Like', dependent: :destroy, foreign_key: 'post_id'
```
## Error when deleting a post
https://user-images.githubusercontent.com/16392046/218262624-5a627834-494e-4e28-b0ed-7691b4c712db.mp4

Reply:
Noted! Thank you so much for the suggestion.